scripts/trias/run_trias.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+
+set -e
+
+# 0 check
+## check input parameters
+if [ $# -ne 2 ]; then
+	echo "usage: ./run_trias.sh GITHUB_USER_NAME GITHUB_PASSWORD"
+	echo
+	echo "As it will clone some repositories from github."
+	exit -1
+else
+	USER_NAME=$1
+	PASSWD=$2
+fi
+
+## check if the user is root
+if [ "$USER" != "root" ]; then
+	echo "This script should be run by root"
+	exit -1
+fi
+
+## check ubuntu version
+lsb_release -a  | grep "Ubuntu 14.04.5"
```

Comment:
my ubuntu version just don't have this command

Reply:
How about 'cat /etc/lsb-release'? can you find this file?

Reply:
Or 'cat /etc/*release'

Reply:
'cat /etc/lsb-release' is ok.